### PR TITLE
Added jquery-ui autocomplete

### PR DESCRIPTION
**Pull request by egonm12**
Deleted the old autocomplete and added the jquery-ui-autocomplete. Tested with Chrome, Safari and Firefox

### CODE REVIEW

File: app/assets/javascripts/jar.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function startLoading(bp) {
     worker = setInterval(increment , 60);
 }
 function stopLoading(bp) {
-    console.log("stoploading")
+    console.log("stoploading");
```
Comment (thnukid) [inline app/assets/javascripts/jar.js]: remove debug info

File: app/assets/javascripts/jar.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ function startLoading(bp) {
...
     clearInterval(worker);
     setText(bp)
```
Comment (thnukid) [inline app/assets/javascripts/jar.js]: missing `;`

File: app/assets/javascripts/transactions.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+jQuery(function() {
+    return $('#transaction_receiver_name').autocomplete({
+        source: $('#transaction_receiver_name').data('autocomplete-source'),
+        autoFocus: true,
+        maxShowItems: 5
+    });
+
```
Comment (thnukid) [inline app/assets/javascripts/transactions.js]: extra blank line

File: app/assets/javascripts/jar.js (hunk 2 above)
Comment (thnukid) [inline app/assets/javascripts/jar.js]: check for javascript linting: https://github.com/douglascrockford/JSLint
There are integration for most editors available and you get warned when doing something rare

File: config/routes.rb [hunk 4]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  Rails.application.routes.url_helpers
```
Comment (thnukid) [inline config/routes.rb]: this look suspecious - for what are you needing the `url_helpers` for?

Comment (egonm12) [inline config/routes.rb]: @thnukid This should be deleted in this branch. It is used in the service slack_notifications.rb in another branch

**Pull request merged**